components/script/dom/htmlformelement.rs

```diff
@@ -106,7 +106,7 @@ impl HTMLFormElementMethods for HTMLFormElement {
     make_setter!(SetAction, "action");
 
     // https://html.spec.whatwg.org/multipage/#dom-form-autocomplete
```

Comment:
While at it, could you change that link to the [correct](https://html.spec.whatwg.org/multipage/#attr-form-autocomplete) one?